dpnp/dpnp_iface_manipulation.py

```diff
@@ -1414,20 +1506,37 @@ def unique(ar, **kwargs):
     return call_origin(numpy.unique, ar, **kwargs)
 
 
-def vstack(tup):
+def vstack(tup, *, dtype=None, casting="same_kind"):
```

Comment:
Could you remove `tests/test_arraymanipulation.py::TestVstack::test_generator` from skipped tests table and modify the test accordingly to be passed?

Could you also move `tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1[lambda x: dpnp.vstack([x, x]).T]` to skipped test table for gpu_no_f64?